README.md

```diff
@@ -412,6 +412,7 @@ Intel RDT flags:
 
 User flags:
 - :whale: :blue_square: `-u, --user`: Username or UID (format: <name|uid>[:<group|gid>])
+- :nerd_face: `--umask`: Set the umask inside the container. Defaults to 0022)
```

Comment:
This seems compatible with Podman, so please add "Corresponds to Podman CLI." as in other Podman-compatible flags such as `--rootfs`.

Eventually we might want to use a different emoji for Podman-compatible options though.